doc/manuals/admin/perf_tuning.md

```diff
@@ -112,6 +113,25 @@ on [the `notifQueue` block](statistics.md#notifqueue-block) may help you to tune
 
 [Top](#top)
 
+## Payload and message size and performance
+
+Orion Context Broker uses two defaults limits related with payloads and HTTP message size. In particular:
+
+* There is a default limit of 1MB in incoming HTTP request payload
+* There is a default limit of 8MB in outgoing HTTP request messages (including HTTP request line, headers and payload), which applies to notifications and forwarded requests
+
+This limit should suffice the most of the use cases and, at the same time, avoids denial of service due to
+too large requests. You can change these limits using the following [CLI flags](cli.md):
+
+* `-inReqPayloadMaxSize` (in bytes) to change de limit in incoming HTTP request payload
+* `-outReqMsgMaxSize` (in bytes) to change the limit for outgoing HTTP request messages
+
+Decreasing the limits could have a positive impact on performance, but may impose limitations in
+Context Broker requests. Increase the limits may have a negative impact in performance, but will allow bigger
```

Comment:
Increase => Increasing

Reply:
Fixed in e2d530b5c16a6a05f1f5c6d807f6cd33a3c76563